content/manuals/docker-hub/download-rate-limit.md

```diff
@@ -98,6 +98,26 @@ both individuals and organizations.
    - [Enforce sign-in](/security/for-admins/enforce-sign-in/) to ensure that you
      can monitor the usage of your users and users receive higher usage limits.
 
+## Pull attribution
+
+Pulls can be attributed to either a personal or organization [namespace](https://docs.docker.com/contribute/style/terminology/#namespace)
+
+**Private Pulls**
```

Comment:
Suggest changing these to a title or adding a colon after because they generate inline with the text below them. 
```suggestion
### Private pulls
```